webapp/src/components/modals/create_update_issue/create_update_issue.jsx

```diff
@@ -46,36 +48,53 @@ export default class CreateOrUpdateIssueModal extends PureComponent {
         this.validator = new Validator();
     }
 
+    getIssueInfo = async () => {
+        const issueInfo = await this.props.issueInfo(this.props.messageData);
+        return issueInfo;
+    }
+
+    updateState(issueInfo) {
+        var {channel_id, title, description, assignees, labels, milestone_title, milestone_number, repo_full_name} = issueInfo ?? {};
```

Comment:
are you updating any of these variable if not use const

Reply:
Use `const`.